sympy/core/numbers.py

```diff
@@ -2681,7 +2681,7 @@ def __new__(cls, expr, coeffs=None, alias=None, **args):
         ...                  [-1, -1, 0, 0], alias=zeta)
         >>> a5.as_poly().as_expr()
         -zeta**3 - zeta**2
-        >>> a5.evalf()
+        >>> a5.evalf(chop=True)
         1.61803398874989
```

Comment:
Why is this change needed?

Reply:
Another good question. It started printing as

```
1.61803398874989 + 0.e-20*I
```

and I don't really understand why. Maybe the `0.e-20*I` used to be eliminated by the very same cancelation I'm trying to get rid of.